biz.aQute.bndlib/src/aQute/bnd/exporter/runbundles/RunbundlesExporter.java

```diff
@@ -33,23 +36,35 @@ public RunbundlesExporter() {}
 	public Entry<String, Resource> export(String type, Project project, Map<String, String> options) throws Exception {
 		project.prepare();
 		Collection<Container> runbundles = project.getRunbundles();
-		Jar jar = new Jar(project.getName());
+		String prefix = options.getOrDefault(TARGET_DIR, "");
+		if (!prefix.endsWith("/")) {
+			prefix = prefix.concat("/");
+		}
+		String template = options.get(TEMPLATE);
+
+		Jar jar = null;
```

Comment:
You don't need to initialize to null since you set jar in both sides of the if.
```suggestion
		Jar jar;
```

Reply:
done